src/applications/vaos/appointment-list/components/AppointmentListNavigation.jsx

```diff
@@ -46,55 +20,53 @@ export default function AppointmentListNavigation({ count, callback }) {
       >
         <ul>
           <li>
-            <button
+            <NavLink
               id="upcoming"
-              type="button"
-              className="va-button-link"
-              onClick={handleClick({
-                history,
-                callback,
-              })}
+              to="/"
+              onClick={() => callback(true)}
               // eslint-disable-next-line jsx-a11y/aria-proptypes
               aria-current={`${
-                location.pathname.endsWith('/') ? 'page' : null
+                location.pathname.endsWith('/') ? 'true' : false
               }`}
             >
               Upcoming
-            </button>
+            </NavLink>
           </li>
           <li>
-            <button
+            <NavLink
               id="pending"
-              type="button"
-              className="va-button-link"
-              onClick={handleClick({
-                history,
-                callback,
-              })}
+              to="/pending"
+              onClick={() => {
+                callback(true);
+                recordEvent({
+                  event: `${GA_PREFIX}-status-pending-link-clicked`,
+                });
+              }}
               // eslint-disable-next-line jsx-a11y/aria-proptypes
               aria-current={`${
-                location.pathname.endsWith('pending') ? 'page' : null
+                location.pathname.endsWith('pending') ? 'true' : false
```

Comment:
Curious why is true set using single quote while false does not?

Reply:
Good catch, once again!